cloudstate-operator/pkg/apis/v1alpha1/statefulstore_types.go

```diff
@@ -82,6 +86,65 @@ type PostgresCredentials struct {
 	DatabaseKey string `json:"databaseKey,omitempty"`
 }
 
+// PostgressSSL is the configuration for Postgres SSL connections.
+type PostgresSSL struct {
+	// SslMode is the mode that should be used for SSL. Valid values are disable, allow, prefer, require,
+	// verify-ca and verify-full. Defaults to disable.
+	// +optional
+	Mode PostgresSSLMode `json:"mode,omitempty"`
+
+	// Secret is the secret that contain any SSL certificates or keys.
+	// +optional
+	Secret *corev1.LocalObjectReference `json:"secret,omitempty"`
+
+	// SSLRootCert is the key of the root certificate in the secret. This must be a PEM encoded X509 certificate.
```

Comment:
Fields do not match comments: RootCert <=> SSLRootCert
Are those comments generated from the CRD YAML files?

Reply:
No, it's the other way around, the CRD YAML files use these comments.